CMakeLists.txt

```diff
@@ -52,6 +52,13 @@ elseif(UNIX)
 endif()
 set(SECP256K1_ROOT "/usr/local")
 
+if(APPLE)
+  set(OPENSSL_ROOT "/usr/local/opt/openssl")
+elseif(UNIX)
+  set(OPENSSL_ROOT "/usr/include/openssl")
+endif()
+set(SECP256K1_ROOT "/usr/local")
+
```

Comment:
as of libtester 5.0 these aren't needed, since #109 moves to libtester 5.0 I wouldn't go through the trouble of adding them

Reply:
This section was in twice. Same in reference-contract. I will remove all references.